src/test/resources/h2-shared-db-application.conf

```diff
@@ -48,6 +48,11 @@ akka-persistence-jdbc {
 
 jdbc-journal {
   use-shared-db = "slick"
+
+  # explicitly setting a lower batchSize.
+  # the test that uses that config is sending 10000 messages per actor
+  # and it fails in H2 when the batch is too large
+  batchSize = 50
```

Comment:
We don't know if this text is speaking the truth. If Travis say so, we can leave like that. Otherwise, I revert it. :-)